tests/zfs-tests/tests/functional/delegate/zfs_allow_003_pos.ksh

```diff
@@ -66,6 +66,10 @@ done
 
 log_must restore_root_datasets
 
+if is_freebsd; then
+	log_must add_group $STAFF_GROUP
+fi
```

Comment:
Why did this group need to be added only on FreeBSD?   It's expected on Linux as well.  And if it is needed why wasn't it removed?

Reply:
This must have been a mistake, the group is added in the delegate setup.